apps/rush-lib/src/schemas/version-policies.schema.json

```diff
@@ -77,6 +77,10 @@
         "exemptFromRushChange": {
           "description": "If true, the version policy will not require changelog files.",
           "type": "boolean"
+        },
+        "includeEmailInChange": {
+          "description": "If true, generated changelog files will include the author's email address.",
```

Comment:
Should we call this setting `includeEmailInChangeFile`?

I think of the terminology like:
- "change" - the work that a developer committed to Git
- "change file" - the JSON files written by the `rush change` command
- "change log" - the CHANGELOG.md human-readable report that is generated by `rush version` during publishing

Reply:
I'm good with changing to `includeEmailInChangeFile`